Compiling indicator-examples failed + fix

**Lightweight Charts™ Version:** <!-- 1.0.0 --> 5.0.8

**Steps/code to reproduce:**

Enter ./indicator-examples

type:
npm run compile

<!--
```js
```
-->

**Actual behavior:**

<!-- A clear and concise description of what actually happen. -->

Compilation fails with many errors like:

Error generating typings for:  ratio-calculation
error TS6053: File 'typings/indicator-examples/src/indicators/ratio/ratio.d.ts' not found.
  The file is in the program because:
    Root file specified for compilation


**Expected behavior:**

<!-- A clear and concise description of what you expected to happen. -->
All files should be compiled.

**Fixed on my local machine:**

In compile.mjs change line 148:
filePath: `./typings/indicator-examples/src/indicators/${stripCalculation(file.exportName)}/${file.exportName}.d.ts`,

to:
filePath: `./typings/indicators/${stripCalculation(file.exportName)}/${file.exportName}.d.ts`,

--- a/indicator-examples/compile.mjs
+++ b/indicator-examples/compile.mjs
@@ -145,7 +145,7 @@ const compile = async () => {
 			}
 			const esModuleTyping = generateDtsBundle([
 				{
-					filePath: `./typings/indicator-examples/src/indicators/${stripCalculation(
+					filePath: `./typings/indicators/${stripCalculation(
 						file.exportName
 					)}/${file.exportName}.d.ts`,
 					// libraries: {

